x-pack/legacy/plugins/reporting/export_types/common/execute_job/get_full_urls.ts

```diff
@@ -40,12 +38,12 @@ export async function getFullUrls({
   // PDF and PNG job params put in the url differently
   let relativeUrls: string[] = [];
 
```

Comment:
We've had security issues here, however build should catch any issues.

Reply:
++ since this is populated by the `job` object, it is based on user input: `JobDocPayloadPNG` and `JobDocPayloadPDF` which come from the payload of a user request.